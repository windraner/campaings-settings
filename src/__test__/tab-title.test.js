import React from 'react';
import TabTitle from '../components/main-content-section/tab-block/common/tab-title/tab-title';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('<TabTitle />', () => {
  it('Check for className and text', () => {
    const wrapper = shallow(<TabTitle text="test" />);
    expect(wrapper.find('.campaigns-settings__tab-title').length).toBe(1);
    expect(wrapper.find('.campaigns-settings__tab-title').text()).toBe('test');
  });
  it('matches the snapshot hide', () => {
    const tree = shallow(<TabTitle text="test" />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
