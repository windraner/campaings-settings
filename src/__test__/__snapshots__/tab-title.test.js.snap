// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabTitle /> matches the snapshot hide 1`] = `
<h2
  className="campaigns-settings__tab-title text-center mb-0"
>
  <strong>
    test
  </strong>
</h2>
`;
